trading/exchanges/abstract_exchange.py

```diff
@@ -56,7 +56,7 @@ def get_exchange_manager(self):
         pass
 
     @abstractmethod
-    async def get_recent_trades(self, symbol, limit=50):
+    async def get_recent_trades(self, symbol, limit=50, candle_range=True):
         pass
```

Comment:
This modification should be replaced by using #954

Reply:
Why ? It's not the same purpose: one is a config value, the other depends on method call context

Reply:
There is no reason to impact abstract_exchange for a backtesting feature.

Reply:
There the reason is inheritence.
We need to find a way to differenciate the results of get_recent_trades in order creation and order fill check, this can be done by a param or by calling a different method. Both affect abstract_exchange. 
Now if you have a solution that is only touching exchange simulator, let's go for it but I didn't find any.

Reply:
I agree to your feature proposal but it's a mistake to add a param to an abstract method for a child implementation purpose.